docs/reference/snapshot-api.asciidoc

```diff
@@ -0,0 +1,184 @@
+[[snapshot-restore-api]]
+=== Snapshot Restore API
+++++
+<titleabbrev>Snapshot Restore API</titleabbrev>
+++++
+////
+Restores a snapshot into a running cluster.
+////
+
+Restores a snapshot into a running cluster.
+
+[source,js]
+--------------------------------------------------
+POST /_snapshot/my_backup/snapshot_1/_restore
+--------------------------------------------------
+// CONSOLE
+
+////
+***********************************************************
+////
+
+////
+Use the appropriate heading levels for your book.
+Add anchors for each section.
+FYI: The section titles use attributes in case those terms change.
+////
+
+[[sample-api-request]]
+==== {api-request-title}
+////
+This section show the basic endpoint, without the body or optional parameters.
+Variables should use <...> syntax.
+If an API supports both PUT and POST, include both here.
+////
+
+`PUT /<follower_index>/_ccr/follow`
+
+
+[[sample-api-prereqs]]
+==== {api-prereq-title}
+////
+Optional list of prerequisites.
+
+For example:
+
+* A snapshot of an index created in 5.x can be restored to 6.x. You must...
+* If the {es} {security-features} are enabled, you must have `write`, `monitor`,
+and `manage_follow_index` index privileges...
+////
+
+
+[[sample-api-desc]]
+==== {api-description-title}
+////
+Add a more detailed description the context.
+Link to related APIs if appropriate.
+
+Guidelines for parameter documentation
+***************************************
+* Use a definition list.
+* End each definition with a period.
+* Include whether the parameter is Optional or Required and the data type.
+* Include default values as the last sentence of the first paragraph.
+* Include a range of valid values, if applicable.
+* If the parameter requires a specific delimiter for multiple values, say so.
+* If the parameter supports wildcards, ditto.
+* For large or nested objects, consider linking to a separate definition list.
+***************************************
+////
```

Comment:
Omit the comments and replace with a longer form description of the API and it's intended usage.